Sources_v3/WebSocketClient/EventMiddlewares/ChannelReadUpdaterMiddleware.swift

```diff
@@ -11,15 +11,9 @@ struct ChannelReadUpdaterMiddleware<ExtraData: ExtraDataTypes>: EventMiddleware
     func handle(event: Event, completion: @escaping (Event?) -> Void) {
         if let event = event as? MessageReadEvent<ExtraData> {
```

Comment:
After the rebase to `master` some tests related to exposing connection events were failing. 

The root cause was in `ChannelReadUpdaterMiddleware` that was blocking events that didn't enter any if statement.

FYI @b-onc @VojtaStavik